election-day/manferna/election_day.go

```diff
@@ -0,0 +1,52 @@
+package electionday
+
+import "fmt"
+
+// NewVoteCounter returns a new vote counter with
+// a given number of initial votes.
+func NewVoteCounter(initialVotes int) *int {
+	var pointerToInitialVotes *int
```

Comment:
You do not need an extra variable here, just return the address of initialVotes (`return &initialVotes`).

On a side note a function which does not have a pointer passed as a parameter, but instead returns a pointer does not makes any sense (not your fault). initialVotes and pointerToInitialVotes are both local to the function, put on the stack and garbage collected once the function returns. It means that external variable holds a pointer to memory marked as free and which can be overwritten.